public_desktop/preload.js

```diff
@@ -1,37 +1,47 @@
-
 const { ipcRenderer, remote } = require( 'electron' );
 const { SpellCheckHandler, ContextMenuListener, ContextMenuBuilder } = require( 'electron-spellchecker' );
+const { Observable } = require( 'rxjs/Observable' );
+require( 'rxjs/add/observable/from' );
```

Comment:
it would be best to add an explanatory comment here why we need to include this and why we're including it for its side-effects

Reply:
@dmsnell Thanks for the review!

There's a fairly detailed explanatory note regarding the use of RxJS a few lines down into the file. I think this should be sufficient? However let me know if I can add or clarify any details.

https://github.com/Automattic/wp-desktop/pull/749/files#diff-c3c73cf27ad747e5e2d364fc3bb986b9R30

> I'm not seeing the mechanism for how the dummy$ gets its input events but I assume that is working anyway?

Yep - it's working. 😅Seems adequate to initialize the observable as empty ( using empty array `[ ]` ) to bypass the `null` check in the spellchecker source code.

Reply:
> There's a fairly detailed explanatory note regarding the use of RxJS a few lines down into the file

we won't suffer by explaining it here either. us the one you're referring to the one in the new line 31? the one that doesn't mention `rxjs` 😉 ?

we don't need a big explanation but coupling the explanation with the line itself will help prevent someone from removing it in the future. in any case, I don't see an explanation anywhere for why we need a side-effect-only import, especially since in the line above we're importing a named object

```js
require('rxjs/add/observable/from'); // necessary for providing Observable.from() to Observable
```